src/test/java/stepDefinition/LoginStepDefinitions.java

```diff
@@ -72,5 +75,32 @@ public void user_isNot_LoggedIn() {
         String expectedURL = "https://qa.koel.app/";
         Assert.assertEquals(driver.getCurrentUrl(), expectedURL);
     }
-}
+
+        @When("I navigate to AllSongs")
+        public void navigateToAllSongs() {
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li a.songs"))).click();
+            WebDriverManager.chromedriver().setup();
+        }
+
+        @And("Context click on first song")
+        public void contextClickOnFirstSong() {
+
+            WebElement firstSongElement = wait.until(ExpectedConditions.visibilityOfElementLocated
+                    (By.cssSelector(".all-songs tr.song-item:nth-child(1)")));
+            actions.contextClick(firstSongElement).perform();
+        }
+
+        @And("Choose Play Option")
+        public void choosePlayOption() {
+            wait.until(ExpectedConditions.visibilityOfElementLocated
+                    (By.cssSelector("li.playback"))).click();
+        }
+
+        @Then("Song is playing")
+        public boolean songIsPlaying() {
+            WebElement soundBarVisualizer = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[data-testid = 'sound-bar-play']")));
+            return soundBarVisualizer.isDisplayed();
+        }
+    }
+
```

Comment:
Add 

```
   @After
    public void closeBrowser(){
        driver.manage().deleteAllCookies();
		driver.quit();
		driver = null;
    }
```

Browsers should be closed after tests

![image](https://github.com/KristinaSarkisyan/A59-QA-Automation-Framework/assets/104761482/a3797d3a-e3b0-4045-a844-71a795fd10d2)